activemq-unit-tests/src/test/java/org/apache/activemq/broker/virtual/VirtualTopicSelectorTest.java

```diff
@@ -99,7 +103,25 @@ protected BrokerService createBroker() throws Exception {
         virtualTopic.setSelectorAware(true);
         VirtualDestinationInterceptor interceptor = new VirtualDestinationInterceptor();
         interceptor.setVirtualDestinations(new VirtualDestination[]{virtualTopic});
-        broker.setDestinationInterceptors(new DestinationInterceptor[]{interceptor});
+        TestDestinationInterceptor testInterceptor = new TestDestinationInterceptor();
```

Comment:
Instead of modifying the existing test, please add a second test to cover the new use case. Changing the test removes the test for the out-of-the-box default behavior vs the custom behavior this fix solves.

Reply:
@Nikita-Shupletsov do you mind to add a new test as proposed by @mattrpav ? I would like to include this PR in ActiveMQ 6.1.4 release.
Thanks !

Reply:
It looks good to me. Thanks !

Reply:
We could have create an abstract but it's really cosmetic, let's go like this for now.